src/components/Sidebar.tsx

```diff
@@ -30,7 +30,13 @@ export const Sidebar = (): JSX.Element => {
   const offersLink = useOffersLink()
 
   return (
-    <nav role="navigation" style={{ minWidth: '362px' }}>
+    <nav
+      role="navigation"
+      style={{
+        width: '320px',
+        borderRight: '1px solid rgba(29, 33, 42, 0.12)'
```

Comment:
couleur en dur ici. utiliser une var cf https://github.com/cozy/cozy-ui/blob/master/react/MuiCozyTheme/makePalette.js ou https://docs.cozy.io/cozy-ui/styleguide/section-settings.html#kssref-settings-theme

Voir dans la maquette quelque chose comme ça
![image](https://user-images.githubusercontent.com/67680939/192725189-ff78c75c-283f-494b-a807-705ba8c34e4b.png)
le `Light/success/main` correspond à `theme.sucess.main` dans `makePalette`

Reply:
- [ ] Find the name of that color somewhere in the theme and update accordingly